openlibrary/data/dump.py

```diff
@@ -50,7 +50,7 @@ def print_dump(json_records, filter=None, print=print):
         if key.startswith(("/b/", "/scan", "/old/")) or not key.startswith("/"):
             continue
 
-        if filter and filter(d) is False:
+        if filter and not filter(d):
```

Comment:
This is a slight change in behaviour ; previously if the filter fn returned `None`, it would not be filtered. So opting to leave that the same in case it's important!
```suggestion
        if filter and filter(d) is False:
```

Reply:
The only usage of the filtering parameter is line 118 of the same file. That lambda treats None as False , hence no need to worry about the None case